src/components/Markdown.jsx

```diff
@@ -553,7 +548,8 @@ function markdownRenderers(theme, classes) {
       </StyledTableCell>
     ),
     image: props => <GImage filename={props.src} alt={props.alt} />,
-    heading: props => HeadingLevelToComponent(props.level, props),
+    heading: props =>
+      HeadingLevelToComponent(props.level, props, previousHeadings),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor. (new-cap)](https://app.codacy.com/app/davwheat/GCSE-Revision-Website/pullRequest?prid=4583220)